spec/support/security_specs_helper.rb

```diff
@@ -106,29 +102,15 @@ def mock_role
   end
 end
 
-shared_examples_for "raises an error when security validation fails" do
-  context 'when security validation fails' do
-    let (:audit_success) { false }
-
-    it 'raises an error' do
-      allow(mocked_security_validator).to(
-        receive(:call).and_raise('FAKE_SECURITY_ERROR')
-      )
-
-      expect { subject }.to raise_error( /FAKE_SECURITY_ERROR/ )
-    end
-  end
-end
-
 shared_examples_for "raises an error when origin validation fails" do
   context 'when origin validation fails' do
     let(:audit_success) { false }
-    
+
     it "raises an error" do
       allow(mocked_origin_validator).to receive(:call)
                                           .and_raise('FAKE_ORIGIN_ERROR')
 
-      expect { subject }.to raise_error( /FAKE_ORIGIN_ERROR/ )
+      expect { subject }.to raise_error(/FAKE_ORIGIN_ERROR/)
```

Comment:
Same comment here about choose a style of a mock or `FAKE_ORIGIN_ERROR`

Reply:
Done